### DISCUSSION TITLE
Icon request: fa-wooden-crate

### DISCUSSION THREAD

**Discussion by stijnvanbael** [FortAwesome/Font-Awesome#21218] [Icon Requests]
A wooden crate, something like this

![169010-200](https://cloud.githubusercontent.com/assets/1050148/23750765/c335574e-04ce-11e7-9524-39f1a39bb851.png)

**Comment by tagliala** [comment]
Is this a duplicate of #10598?

**Comment by DigitalAlan** [comment]
I looked at 10598 just mho - but that is a jail cell with a door

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
